.github/dependabot.yml

```diff
@@ -11,26 +11,30 @@ updates:
       - 'type: dependencies'
       - 'type: automerge'
     allow:
-      - dependency-type: development
+      - dependency-type: direct
     ignore:
       - dependency-name: react
         versions:
-          - '^16.11.0'
+          - '>=18.0.0'
       - dependency-name: react-dom
         versions:
-          - '^16.11.0'
+          - '>=18.0.0'
+      - dependency-name: react-test-renderer
+        versions:
+          - '>=18.0.0'
       - dependency-name: '@types/react'
         versions:
-          - '^16.10.0'
+          - '>=18.0.0'
       - dependency-name: '@types/react-dom'
         versions:
-          - '^16.10.0'
+          - '>=18.0.0'
       - dependency-name: '@storybook/react'
         versions:
           - '^6.0.21'
```

Comment:
Am I reading this right that we are ignoring versions of `@storybook/react` that are compatible with `6.0.21`? Same for the `addon-essentials` below? I'm pretty sure I'm the one that added those incorrectly.

Reply:
[Based on the docs](https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#ignore), this means we will ignore all versions of `@storybook/react` between `6.0.21` and `7.0.0`. I imagine we probably want the inverse of that? That's why I changed our React version to a `>=` syntax since we are okay with minor and incremental version changes, but we want to ignore major version changes. We just have to remember to update these ignores for every major version migration if we want to keep getting minor updates.

Reply:
Yeah, that's what I think, too. I think I misunderstood when I originally set it up. Should we go ahead and fix everything at the same time?

Reply:
That's perfectly reasonable! I know storybook tends to be a bit more brittle, so if we do run into issues with minor version updates after this PR, we can always re-evaluate.

Reply:
2646e3a